roles/test-provision-unifiedpush-apb/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+- name: Check that the unifiedpush login webpage is accessible
+  uri:
+    url: "https://{{ ups_route.route.spec.host }}"
```

Comment:
should we perhaps also test that a PushAPP was created ?

Reply:
test added